graph-service/enricher.py

```
@@ -0,0 +1,909 @@
+import os
+import asyncio
+import logging
+import json
+import time
+import schedule
+import aiohttp
+import urllib.parse
+import re
+import html as html_stdlib
+from typing import List, Dict, Optional, Any
```

Ruff (0.14.7)

`typing.List` is deprecated, use `list` instead

(UP035)